reactor-core/src/main/java/reactor/core/publisher/FluxReplay.java

```diff
@@ -1146,6 +1146,7 @@ public Object scanUnsafe(Scannable.Attr key) {
 		if (key == Attr.PREFETCH) return getPrefetch();
 		if (key == Attr.PARENT) return source;
 		if (key == Attr.RUN_ON) return scheduler;
+		if (key == Attr.RUN_STYLE) return Attr.RunStyle.ASYNC;
```

Comment:
is it really `ASYNC`? I see it uses `Scheduler` for getting time

Reply:
Fixed in commit [abd4003](https://github.com/reactor/reactor-core/pull/2123/commits/abd40036e30e787e47e07fc7461c90fbdb1102e0)